
  0%|                                                                                                                                                                                   | 0/5460 [00:00<?, ?it/s][WARNING|logging.py:295] 2023-07-16 00:37:52,431 >> `use_cache=True` is incompatible with gradient checkpointing. Setting `use_cache=False`...































































































































































































































































































































































































































































































































































 10%|████████████████▋                                                                                                                                                      | 546/5460 [26:08<2:50:18,  2.08s/it]
  6%|█████████▉                                                                                                                                                                 | 17/292 [00:00<00:16, 16.92it/s]










  warnings.warn(
{'eval_loss': 2.673828125, 'eval_accuracy': 0.5065414806833923, 'eval_runtime': 20.0129, 'eval_samples_per_second': 14.591, 'eval_steps_per_second': 14.591, 'epoch': 1.0}































































































































































































































































































































































































































































































































































 20%|█████████████████████████████████▏                                                                                                                                    | 1092/5460 [52:39<2:31:24,  2.08s/it]
  1%|█▊                                                                                                                                                                          | 3/292 [00:00<00:11, 24.97it/s]









100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████▍| 291/292 [00:17<00:00, 16.53it/s]

  warnings.warn(































































































































































































































































































































































































































































































































































 30%|█████████████████████████████████████████████████▏                                                                                                                  | 1636/5460 [1:19:06<3:03:09,  2.87s/it]
 30%|█████████████████████████████████████████████████▏                                                                                                                  | 1638/5460 [1:19:09<2:12:30,  2.08s/it]








100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████▍| 291/292 [00:17<00:00, 16.58it/s]
  warnings.warn(
Traceback (most recent call last):
  File "/home/eli.richmond/miniconda3/envs/py310/lib/python3.10/site-packages/torch/serialization.py", line 441, in save
    _save(obj, opened_zipfile, pickle_module, pickle_protocol)
  File "/home/eli.richmond/miniconda3/envs/py310/lib/python3.10/site-packages/torch/serialization.py", line 668, in _save
    zip_file.write_record(name, storage.data_ptr(), num_bytes)
RuntimeError: [enforce fail at inline_container.cc:471] . PytorchStreamWriter failed writing file data/0: file write failed
During handling of the above exception, another exception occurred:
Traceback (most recent call last):
  File "/home/eli.richmond/reu/train_gptj_toolformer.py", line 702, in <module>
    main()
  File "/home/eli.richmond/reu/train_gptj_toolformer.py", line 639, in main
    train_result = trainer.train(resume_from_checkpoint=checkpoint)
  File "/home/eli.richmond/miniconda3/envs/py310/lib/python3.10/site-packages/transformers/trainer.py", line 1664, in train
    return inner_training_loop(